Source/Public/Update-BicepParameterFile.ps1

```diff
@@ -0,0 +1,75 @@
+function Update-BicepParameterFile {
+    [CmdletBinding()]
+    param (
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory, Position = 1)]
+        [string]$Path,
```

Comment:
Can we add a validate script to `-Path` to only accept `.json` files as input. Accidentally pointed to the `.bicep` file while testing and it broke my template.